doc/classes/@GlobalScope.xml

```diff
@@ -1273,6 +1273,9 @@
 		<constant name="INLINE_ALIGNMENT_TEXT_MASK" value="12" enum="InlineAlignment">
 			A bit mask for [code]INLINE_ALIGNMENT_TO_*[/code] alignment constants.
 		</constant>
+		<constant name="KEY_NONE" value="0" enum="Key">
+			Enum value when no key.
```

Comment:
That sounds pretty awkward.
How about:
```suggestion
			Enum value which doesn't correspond to any key. This is used to initialize [Key] properties with a generic state.
```
Same for others.

Reply:
Far better, will change this